src/django_rich/runner.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import sys
+import unittest
+from unittest.result import STDERR_LINE, STDOUT_LINE
+
+from django.test import testcases
+from django.test.runner import DebugSQLTextTestResult, DiscoverRunner, PDBDebugResult
+from rich.color import Color
+from rich.console import Console
+from rich.rule import Rule
+from rich.style import Style
+from rich.traceback import Traceback
+
+
+class RichMixin(unittest.TextTestResult):
```

Comment:
I'm not sure we should inheit from `TextTestResult` if it ends up with just lots of `super(...)` hacks to skip it in inheritance. Better to inherit from the base class `TestResult` and then copy in the behaviour from `TextTestResult` that we need - which may not be anything?